content/the-road-to-ember-4-0.md

```diff
@@ -0,0 +1,89 @@
+
+Read more about this change in [the deprecation guide](https://deprecations.emberjs.com/v3.x/#toc_3-0-browser-support-policy) and at [Ember's browser support policy page](https://emberjs.com/browser-support/).
+
+Existing Ember users should note that Ember 3.27 has already removed IE11 from the default target list for production and testing builds.
+
+### Planning your upgrade to 4.0
+
+For each API removed in Ember 4.0 you can find an entry in the [Ember 3.x deprecation guide](https://deprecations.emberjs.com/v3.x/). In many applications, you have probably addressed many of these warnings already.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon as part of your upgrade process. The workflow addon allows you to create a configuration file to silence most deprecations, allowing you to focus on one or a few warnings at a time. If you're working with a large codebase and many contributors, the focused control of deprecations will also allow you to throw when deprecations you've already addressed are re-introduced, preventing backsliding in your upgrade process.
+
+In contrast to prior major releases, we have no plans to offer an
+`ember-3-legacy` package which makes Ember 3.x APIs available for a limited time
+in Ember 4.0. This is based on fairly low use of these packages in the past.
+
+**Don't panic.** We expect many Ember users to be enthusiastic to adopt our 4.0 release, but for conservative consumers we continue to suggest using Ember's Long-Term Support releases. Ember 3.28, the last version of the 3.x cycle, will become an LTS in September 2021. The first 4.x release to be promoted to LTS will be Ember 4.4 around March 2022. This timeline provides a long window for your business to address any remaining 4.0-targeted deprecations if you're on the LTS cycle.
+
+### Contributing to Ember 4.0
+
+Across [Ember.js](https://github.com/emberjs/ember.js), [Ember Data](https://github.com/emberjs/data), and [Ember CLI](https://github.com/ember-cli/ember-cli) repos the final beta cycle of 3.x has already started. The `master` branch of each repo is ready for contribution to remove already deprecated functionality and IE11 compatibility code.
+
+The core framework effort to prepare for 4.0 is tracked at
+[emberjs/ember.js#19545](https://github.com/emberjs/ember.js/issues/19545), and
+a list of deprecated APIs we need help to remove is tracked at
+[emberjs/ember.js#19617](https://github.com/emberjs/ember.js/issues/19617).
+
+Join us on [Discord](https://discord.com/invite/emberjs) in the `#dev-ember-js`, `#dev-ember-data`, or `dev-ember-cli` channels to find out how you can contribute.
+
+In less than six weeks we expect the first betas for 4.0 to be released. Please plan to test those releases with your applications and addons to smoke out any regressions.
```

Comment:
```suggestion
In less than six weeks, we expect the first betas for 4.0 to be released. Please help test the betas on your applications and addons so that we can catch any regressions.
```